sport/app/cricket/feed/CricketThrottler.scala

```diff
@@ -19,8 +20,8 @@ class CricketThrottlerActor()(implicit materializer: Materializer) extends Actor
 
   private case class TaskWithSender[+T](sender: ActorRef, task: () => Future[T])
 
-  val completionMatcher: PartialFunction[Any, CompletionStrategy] = { case e => CompletionStrategy.immediately }
-  val failureMatcher: PartialFunction[Any, Throwable] = { case e => new Throwable() }
+  val completionMatcher: PartialFunction[Any, CompletionStrategy] = { case Success => CompletionStrategy.draining }
```

Comment:
The change to `.draining` here is not that important - we don't ever complete the stream explicitly so the strategy chosen is a bit redundant. However `draining` is the default and probably a sensible choice for our stream, which would make sense  to drain of existing messages.